qa/api/src/test/java/com/epam/cm/tests/ForgotPasswordTests.java

```diff
@@ -0,0 +1,54 @@
+package com.epam.cm.tests;
+
+import com.epam.cm.base.SimpleBaseTest;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.junit.Test;
+
+import static io.restassured.RestAssured.given;
+
+/**
+ * Created by Mariia_Koltsova on 10/6/2017.
+ */
+public class ForgotPasswordTests extends SimpleBaseTest {
+
+    @Test
+    public void positiveForgotPasswordTest(){
+        //Response response1 =
+                given()
+                        .contentType(ContentType.JSON)
+                        .baseUri(config.baseHost)
+                        .auth().preemptive().basic(config.speakerUser, config.speakerPassword)
+                        .cookie("XSRF-TOKEN", response.cookie("XSRF-TOKEN"))
+                        .header("X-XSRF-TOKEN", response.cookie("XSRF-TOKEN"))
+                        .body("{\"mail\":\"test@test.com\"}")
+                        .
+                                when()
+                        .post( "/api/forgot-password")
```

Comment:
Introduce the constant for endpoint URL = "/api/forgot-password"; and use this variable within each test. 
Better introduce new class EndpointUrl and collect of URL with this class